.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
           - stable
           - beta
           - nightly
-        os: [ubuntu-18.04]
+        os: [ubuntu-22.04]
```

Comment:
IIRC we originally used an old version to make sure stuff still works on the old version. For that  reason I migrated some of the other repos to the oldest version that's still supported rather than the newest version (in this case 20.04).

Reply:
That sounds reasonable. I can amend my branches to prefer 20.04 where we were using 18.04 and leave any existing 20.04's as-is.

Reply:
Sounds good to me.

Reply:
Fixed for this branch with the push to https://github.com/rustls/pemfile/commit/ce7b706c85153a8b0a13c2ce36d9133fda4bdd1f. Slowly working through the other PRs :-)